packages/synchro-charts/src/components/sc-dial/sc-dial.tsx

```diff
@@ -0,0 +1,161 @@
+import { Component, Event, EventEmitter, h, Prop, State } from '@stencil/core';
+import {
+  DataPoint,
+  DataStream,
+  MessageOverrides,
+  MinimalSizeConfig,
+  SizeConfig,
+  StreamAssociation,
+  ViewPortConfig,
+} from '../../utils/dataTypes';
+import { Annotations, DialConfig, Threshold, WidgetConfigurationUpdate } from '../charts/common/types';
+import { validate } from '../common/validator/validate';
+import { NameValue, updateName } from '../sc-data-stream-name/helper';
+import { getDataStreamForEventing } from '../charts/common';
+import { RenderCell } from './type';
+import { breachedThreshold } from '../charts/common/annotations/breachedThreshold';
+import { isMinimalStaticViewport } from '../../utils/predicates';
+import { getThresholds } from '../charts/common/annotations/utils';
+import { RectScrollFixed } from '../../utils/types';
+import { DEFAULT_CHART_CONFIG } from '../charts/sc-webgl-base-chart/chartDefaults';
+
+const renderCell: RenderCell = props => <sc-dial-base {...props} />;
+
+const DEFAULT_MARGINS: Partial<SizeConfig> = {
+  marginLeft: 10,
+  marginTop: 0,
+  marginBottom: DEFAULT_CHART_CONFIG.size.marginBottom,
+  marginRight: 5,
+};
+
+// Fits two rows of legend rows
+const THRESHOLD_LEGEND_HEIGHT_PX = 50;
```

Comment:
there is no threshold - make sure to clean up stray copy and pasted code that does not pertain

Reply:
Remove THRESHOLD_LEGEND_HEIGHT_PX value.